apps/patient/src/views/Pharmacy.tsx

```diff
@@ -54,6 +81,142 @@ const query = (method, data) =>
     });
   });
 
+const geocode = async (address: string) => {
+  const data = await geocoder.geocode({ address });
+  if (data?.results) {
+    return {
+      address: data.results[0].formatted_address,
+      lat: data.results[0].geometry.location.lat(),
+      lng: data.results[0].geometry.location.lng()
+    };
+  } else {
+    throw new Error('Failed to geocode the address.');
```

Comment:
Does google have an error message we can include?

Reply:
statuses, added